gnosisSafe.md

```diff
@@ -0,0 +1,282 @@
+# Gnosis Safe Forwarder
+
+We assume you have set up your nodejs environment and you simply need to plug-in our library.
+
+1. **Install**. You need to install the NPM pacakge:
+
+```
+npm i @anydotcrypto/metatransactions --save-dev
+```
+
+2. **Environment**. Decide which network to use.
+
+You will need to import ChainID into your code:
+
+```
+import { ChainID } from "@anydotcrypto/metatransactions";
+```
+
+Our library currently supports MAINNET or ROPSTEN.
+
+```
+// Blockchain ID (Mainnet = 3)
+ChainID.MAINNET;
+ChainID.ROPSTEN;
+```
+
+The Gnosis Safe library only supports NONCE replay protection. Check out our [wallet contracts](proxyAccounts) for more exotic replay protection.
+
+3. Let's create the meta-transaction library with your preferred options!
+
+To instantiate the Gnosis Safe forwarder:
+
+```js
+const signer = Wallet.Mnemonic("");
```

Comment:
should be:
```
const signer = Wallet.Mnemonic("").connect(provider);
```
maybe check all other places where we new up a wallet

Reply:
fixed